docs/hub/dduf.md

```diff
@@ -0,0 +1,166 @@
+
+We've seeded some DDUFs of popular formats for the community to play with: https://huggingface.co/DDUF, check them out!
+
+<Tip>
+
+We welcome contributions with open arms!
+
+To create a widely adopted file format, we need early feedback from the community. Nothing is set in stone, and we value everyone's input. Is your use case not covered? Please let us know! Discussions about the DDUF format happen in the [DDUF organization](https://huggingface.co/spaces/DDUF/README/discussions/2).
+
+</Tip>
+
+## Motivation
+
+Yet, another file format? Yes, but for good reasons!
+
+The primary goal of DDUF is to create a community-endorsed single-file format for diffusion models. Current model distribution methods often involve multiple separate files, different weight-saving formats, and managing files from various locations. DDUF aims to solve these challenges by packaging all model components into a single file, enforcing a consistent structure while being opinionated about saving formats.
+
+The DDUF format is also designed to be language-agnostic. While we currently provide tooling for the Python ecosystem, there's nothing stopping similar tools from being developed in JavaScript, Rust, C++, and other languages. Like GGUF or safetensors, DDUF is built to be parsable from a remote location without downloading the entire file, which will enable advanced support on the Hugging Face Hub.
+
+## Key Features
+
+1. **Single file** packaging.
+2. Based on **ZIP file format** to leverage existing tooling.
+3. No compression, ensuring **`mmap` compatibility** for fast loading and saving.
+4. **HTTP-friendly**: metadata and file structure can be fetched remotely using HTTP Range requests.
+5. **Flexible**: each model component is stored in its own directory, following the current `diffusers` structure.
+6. **Safe**: uses `safetensors` as weights saving format and prohibits nested directories to prevent ZIP-bombs.
+
+## Technical specifications
+
+Technically, a `.dduf` file **is** a [`.zip` archive](https://en.wikipedia.org/wiki/ZIP_(file_format)). By building on a universally supported file format, we ensure robust tooling already exists. However, some constraints are enforced to meet diffusion models' requirements:
+- Data must be stored uncompressed (flag `0`), allowing mmap-compatibility.
+- Data must be stored using ZIP64 protocol, enabling saving files above 4GB.
+- The archive can only contain `.json`, `.safetensors`, `.model` and `.txt` files.
+- A `model_index.json` file must be present at the root of the archive. It must contain a key-value mapping with metadata about the model and its components.
+- Each component must be stored in its own directory (e.g., `vae/`, `text_encoder/`). Nested files must use UNIX-style path separators (`/`).
+- Each directory must correspond to a component in the `model_index.json` index.
+- Each directory must contain a json config file (one of `config.json`, `tokenizer_config.json`, `image_processor.json`).
```

Comment:
This should also include `scheduler_config.json`. 

For image processors, we should do `preprocessor_config.json` ([example](https://huggingface.co/google/siglip-base-patch16-384/blob/main/preprocessor_config.json)) (~image_processor.json~). Sincere apologies from my side for not being correct in the first place.